neurom/apps/morph_stats.py

```diff
@@ -131,8 +131,10 @@ def _get_feature_stats(feature_name, morphs, modes, kwargs):
 
         stat = value
         if isinstance(value, Sized):
-            if len(value) == 0 and mode not in {'raw', 'sum'}:
+            if len(value) == 0:
                 stat = None
+            elif mode == 'raw':
+                stat = value
```

Comment:
Before NeuroMv3 it was `if mode == raw: return values.tolist()`, so maybe we should have `stat = value.to_list()` here to be consistent?
@asanin-epfl do you remember why you changed this in #950 ?

Reply:
ah yes, I can add .tolist() indeed.

Reply:
.tolist is not needed, `value` is a list already

Reply:
ah ok!

Reply:
Ah ok, it was changed elsewhere. Sorry :)

Reply:
No, I changed I think. I was just bluntly optimizing. I fixed docstrings so now it's clear that feature values can't be numpy arrays. They are either a single number or a list of numbers. I didn't like joggling of values between numpy arrays and lists. So I choose one format for consistency. The simpler one. If user want he can always convert list to a numpy array.

Reply:
Ok, it's simpler like this indeed. Thanks for the clarification.